src/app/register/pages/register/register.component.ts

```diff
@@ -102,7 +102,10 @@ export class RegisterComponent implements OnInit, AfterViewInit {
             this.FormGroupStepA = this.prefillRegisterForm(
               this.platform.queryParameters
             )
-          } else if (session.oauthSession) {
+          } else if (
+            session.oauthSession &&
+            Object.keys(this.platform.queryParameters).length > 0
```

Comment:
Great! 
Just an improvement... use the `platform.hasOauthParameters` here as is the central source of truth about this.